### PR TITLE
Pricing section

### PR DESCRIPTION
**Pull request by Ma9da**
image

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -104,4 +104,132 @@ flex-wrap: wrap;
...
+}
+.section__header{
+    text-align: center;
+}
+.section__title{
+    color: var(--text-color);
+    font-weight: 700;
+    font-size: 2.5rem;
+    text-transform: capitalize;
+    padding-bottom: 27px;
```
Comment (Safei-Ashraf) [inline css/style.css]: rem

File: css/style.css [hunk 2]
```diff
@@ -104,4 +104,132 @@ flex-wrap: wrap;
...
+    color: var(--second-text-color);
+    text-align: center;
+}
+.pricing__packages-header{
+    font-weight: 700;
+}
+.pricing__packages-header h5{
+    color: var(--text-color);
+    font-size: 1.5rem;
+    margin: 35px 0px;
```
Comment (Safei-Ashraf) [inline css/style.css:459]: rem

File: css/style.css [hunk 3]
```diff
@@ -104,4 +104,132 @@ flex-wrap: wrap;
...
+    padding: 15px 40px;
+    border-radius: 5px;
+    -webkit-border-radius: 5px;
+    -moz-border-radius: 5px;
+    -ms-border-radius: 5px;
+    -o-border-radius: 5px;
+}
+
+/***** End pricing *****/
+@media screen and (max-width: 1200px) and (min-width:577px){
```
Comment (Safei-Ashraf) [inline css/style.css]: use one break point threshold

File: index.html [hunk 4]
```diff
@@ -56,6 +56,96 @@ <h3><a href="#">physionic</a></h3>
         </div>
     </header>
     <!-- ========== End hero ========== -->
+    <!-- ========== Start pricing ========== -->
+    <div class="pricing reviews">
```
Comment (Safei-Ashraf) [inline index.html]: section elemnt

**Review by Safei-Ashraf — CHANGES_REQUESTED** [review]
revise all spacing values to use rem instead of px

File: css/style.css (hunk 3 above)
Comment (Ma9da) [reply css/style.css]: the laptop/tablet design is sometimes different than the mobile design

Comment (Safei-Ashraf) [reply css/style.css]: you still can do that with a single breakpoint order.

**Pull request merged**